README.md

```diff
@@ -116,7 +136,7 @@ body, that says "hello".
 
 Your Answer:
 ```js
-// your code here
+  $(".greeting").on("click").append(<p>"hello"</p>);  
 ```
 
```

Comment:
`.on()` takes two arguments: the type of event (e.g., "click") and a callback function that describes what should be done when the event is triggered.

To fix your answer you need to (1) add that callback argument and (2) take care of the appending in the body of that callback function. Also keep in mind what exactly you're appending that paragraph to. You're going to need to attach that `.append()` to a jQuery selector.